src/Data/Nat/Properties.agda

```diff
@@ -657,6 +657,17 @@ i*j≡1⇒j≡1 i j eq = i*j≡1⇒i≡1 j i (trans (*-comm j i) eq)
   ; ε-homo  = refl
   }
 
+^-assoc-^-* : ∀ m n p → (m ^ n) ^ p ≡ m ^ (n * p)
```

Comment:
I think this should probably be called `^-*-assoc` to be consistent with the rest of the "almost" algebraic property proofs? Usually where the operators don't quite match we replace the single operator with both in the standard name, rather than sticking them both at the end.

Reply:
I must admit I had no idea how to name this one so I tried to mimick `^-distribˡ-+-*`.

Reply:
Hmm, yes I understand how both patterns are inferable from that! 

My logic is that say you have some property usually named `X-property-Z` and instead of operator `X` you have operators `X` and `Y` then the name should change to `X-Y-property-Z` (or `X-property-Y-Z` if instead of operator `Z` you have operators `Y` and `Z`). 

Am I making any sense?

Reply:
Ok, that makes sense.